content/c-sharp/concepts/strings/terms/endswith/endswith.md

```diff
@@ -0,0 +1,126 @@
+---
+Title: '.EndsWith()'
+Description: 'Determines if the end of the string instance matches the specified string and returns a boolean.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Characters;
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.EndsWith()`** method determines whether the end of the String instance matches the specified character or string.  Additional parameters may be supplied to define the criteria for a match.
```

Comment:
```suggestion
The **`.EndsWith()`** method determines whether the end of the `String` instance matches the specified character or string.  Additional parameters may be supplied to define the criteria for a match.
```

Reply:
@SSwiniarski I made the change that you suggested and pushed the updated file.  Thanks.